bitnami/kafka/templates/NOTES.txt

```diff
@@ -48,26 +48,39 @@ To start a kafka consumer run the following command:
     kubectl --namespace {{ .Release.Namespace }} exec -it $POD_NAME -- kafka-console-consumer.sh --bootstrap-server localhost:9092 --topic test --from-beginning
     {{- end }}
 
-To connect to your Kafka server from outside the cluster execute the following commands:
+{{- if .Values.externalAccess.enabled }}
 
-{{- if contains "NodePort" .Values.service.type }}
+To connect to your Kafka server from outside the cluster check the following information:
 
-    export NODE_IP=$(kubectl get nodes --namespace {{ .Release.Namespace }} -o jsonpath="{.items[0].status.addresses[0].address}")
-    export NODE_PORT=$(kubectl get --namespace {{ .Release.Namespace }} -o jsonpath="{.spec.ports[0].nodePort}" services {{ template "kafka.fullname" . }})
-    echo "Kafka Broker Endpoint: $NODE_IP:$NODE_PORT"
+{{- if contains "NodePort" .Values.externalAccess.service.type }}
 
-{{- else if contains "LoadBalancer" .Values.service.type }}
+{{- if .Values.externalAccess.service.domain }}
+
+    Kafka brokers domain: Use your provided hostname to reach Kafka brokers, {{ .Values.externalAccess.service.domain }}
+
+{{- else }}
+
+    Kafka brokers domain: You can get the external node ip from the Kafka configuration file with the following command (Check the EXTERNAL listener)
+
+        kubectl exec -it KAFKA_POD -- cat /opt/bitnami/kafka/conf/server.properties | grep advertised.listeners
```

Comment:
What is "KAFKA_POD"? Please add the instructions for users to obtain a random pod instead.

Reply:
updated instructions